frontend/src/i18n/locales/en.yaml

```diff
@@ -302,13 +302,25 @@ manage:
       user: User
       yourself: You
       subset-warning: 'This selection is already included in the following group(s): {{groups}}.'
+      inherited: Inherited permissions
+      inherited-tooltip: >
+        The following permissions were granted in a higher level page and are inherited to all subpages.
       actions:
         title: Actions
         read: Read
         read-description: Can watch the video and view metadata.
         write: Read and write
-        large-group-warning: You are giving write-access to a large group. Is this intentional?
         write-description: Can also edit the video, metadata and other details.
+        moderate: Moderate
+        moderate-description: Can edit content, add subpages and change the name and subpage order.
+        admin: Manage
```

Comment:
As we just discussed in chat: we wanted to user "page admin" for user facing text as well.